plugins/feature/simpleptt/simplepttgui.cpp

```diff
@@ -307,18 +310,23 @@ void SimplePTTGUI::updateDeviceSetLists()
             ui->txDevice->addItem(QString("T%1").arg(deviceIndex), deviceIndex);
             txIndex++;
         }
+        else if (deviceMIMOEngine)
+        {
+            QString text = QString("M%1").arg(deviceIndex);
+            ui->rxDevice->addItem(text, deviceIndex);
```

Comment:
For MIMO devices you also have to specify a stream index as it handles several I/Q streams simultaneously

Reply:
> For MIMO devices you also have to specify a stream index as it handles several I/Q streams simultaneously

In SimplePTT's case, it needs only a switch of Rx/Tx.
Seems that specifying stream indexes is unnecessary here, as the MIMO interface is defined without a stream-specified switch.
https://github.com/f4exb/sdrangel/blob/f35ea35a2dae318c6bef63ef99267accc6548c0d/sdrbase/dsp/devicesamplemimo.h#L60

Reply:
I have to check. It means all Rx or Tx streams start and stop at once or just the ones with index 0. If this is not a bug it is a limitation but maybe not to be addressed here.

Reply:
As far as I see, the current startRx/startTx implementation is switching all streams at once. Maybe an extension is needed for multi-stream devices.

https://github.com/f4exb/sdrangel/blob/f35ea35a2dae318c6bef63ef99267accc6548c0d/sdrbase/dsp/dspdevicemimoengine.cpp#L713

Reply:
In fact this is normal as the different streams are supposed to run synchronously.